engine/env/spark/pom.xml

```diff
@@ -136,6 +136,22 @@
 					<groupId>org.eobjects.metamodel-extras</groupId>
 					<artifactId>MetaModel-extras-access</artifactId>
 				</exclusion>
+                <exclusion>
```

Comment:
Seems ok, with question to the DtaCleaner-xml-config. How to beware compile time dependency to DataCleaner-remote-component... The JaxbConfigurationReader from there builds the components descriptor providers (so possibly the remote components descriptor provider, which must depend on DataCleaner-remote-component (or will be part of it)).

Is there any possibility that the JaxbConfigReader could build the Remote descriptor provider in some IoC style? So if it is on classpath, it uses it, if not, will be not available. And without any compile dependency? I can imagine it e.g. with Spring: the bean is injected by classpath scanning by recognizing an interface. Is something similar available in DataCleaner? If not, would be using bit of reflection here a big evil?